cli/azd/docs/featue-toggle.md

```diff
@@ -0,0 +1,61 @@
+# Feature Toggle
```

Comment:
I think this doc is great enough to be thought of as the `product` spec instead of a `proposal` spec. If you agree, let's decide on the proper noun of the feature.

I would vote on:

- Renaming doc to `experimental-features.md`
- Renaming title to `Experimental Features`
- Concisely define the package name as `expfeature` to support the functionality (I think it warrants a package based on what I'm seeing)